core/src/test/java/io/grpc/util/RoundRobinLoadBalancerTest.java

```diff
@@ -352,11 +352,40 @@ public void nameResolutionErrorWithActiveChannels() throws Exception {
     verifyNoMoreInteractions(mockHelper);
   }
 
+  @Test
+  public void subchannelStateIsolation() throws Exception {
+    final List<Subchannel> createdSubchannels = Lists.newArrayList();
+    when(mockHelper.createSubchannel(any(EquivalentAddressGroup.class), any(Attributes.class)))
+        .then(new Answer<Subchannel>() {
+          @Override
+          public Subchannel answer(InvocationOnMock invocation) throws Throwable {
+            Subchannel sc = createMockSubchannel((Attributes)invocation.getArguments()[1]);
+            createdSubchannels.add(sc);
+            return sc;
+          }
+        });
```

Comment:
Could you merge this change into the stubbing of `createSubchannel()` in `setUp()`?